build/compile_descriptor

```diff
@@ -20,20 +20,26 @@ is_up_to_date () {
 
 set -e
 
+if [ -d "./ebin" ]; then
+    OUT="./ebin"
+else
+    OUT="$REBAR_DEPS_DIR/gpb/ebin"
+fi
+
```

Comment:
But it is a post hook, so the ebin will definitely be there by the time it runs.